Sources/Bot/API/Repository.swift

```diff
@@ -92,6 +92,14 @@ extension Repository {
         )
     }
 
+    func issueComments(in pullRequest: PullRequest) -> Resource<[IssueComment]> {
+        return Resource(
```

Comment:
```suggestion
        Resource(
```

Reply:
I'll check but I'm not entirely sure this is supported on the Swift version we're using on Linux, one sec

Reply:
Ah, that makes sense. Feel free to disregard then.